issue_test.go

```diff
@@ -91,21 +91,23 @@ var _ = Describe("Issue", func() {
 		})
 
 		It("should provide accurate line and file information for multi-line statements", func() {
-			var target *ast.BinaryExpr
```

Comment:
Think the goal of this test was to make sure line range was accurate for findings that span multiple lines like this:

```go
2: 
3: q := `SELECT * FROM table WHERE` +
4:    os.Args[1] +  `= ?`
5:
```
So would return a range `3-4` rather than just `3` for example. The test code you've replaced here doesn't really do that afaict.

Reply:
Thanks for catching this. I changed the test to generate a multi-line finding.